scripts/test-runner.js

```diff
@@ -556,7 +556,6 @@ function run(args) {
     // filter hidden files
     if (path.basename(test.name)[0] === ".") continue;
     if (test.name.endsWith("~")) continue;
-    if (test.file.includes("// skip")) continue;
```

Comment:
Seems like a feature we don't want to get rid off. Perhaps this can become "// skip this test for now".